app/services/fee_scheme.rb

```diff
@@ -0,0 +1,27 @@
+class FeeScheme
+  def initialize(type, date)
+    @type = type
+    @date = date
+  end
+
+
+  def fee_scheme
+    @fee_scheme ||= api_call.fee_schemes(type: @type, case_date: @date)
+  end
+
+  def calculate
+    fee_scheme.calculate(
+      scenario: 5,
+      advocate_type: 'JRALONE',
+      offence_class: 'E',
+      fee_type_code: 'AGFS_APPEAL_CON',
+      day: 1,
+      number_of_defendants: 1,
+      number_of_cases: 1
+    )
+  end
+
+  def api_call
+    @client ||= LAA::FeeCalculator.client
```

Comment:
The instance variable used to memoize the data should match the method name. So either the method should be called `client` or the instance variable should be `@api_call`. I think that `client` is more sensible.